### PR TITLE
Adding resume builder functionality

### PR DESCRIPTION
**Pull request by abhishek-sheth10**
Added resume builder feature to generate resume

### CODE REVIEW

File: backend/test_app.py [hunk 1]
```diff
@@ -32,7 +31,11 @@ def client():
         password = info["password"]
         app.config["MONGODB_SETTINGS"] = {
             "db": "appTracker",
+<<<<<<< Updated upstream
```
Comment (sachindoddaguni) [inline backend/test_app.py:34]: update this pls. include the one from upstream

File: frontend/yarn.lock
Comment (sachindoddaguni) [inline frontend/yarn.lock:1]: Lets not commit this

File: requirements.txt
Comment (sachindoddaguni) [inline requirements.txt:1]: Add version for pyton docx

Comment (sachindoddaguni) [inline requirements.txt:1]: Update the same backend/requirements.txt

**Review by sachindoddaguni — CHANGES_REQUESTED** [review]

File: requirements.txt
Comment (abhishek-sheth10) [reply requirements.txt:1]: updated

**Review by sachindoddaguni — APPROVED** [review]

**Pull request merged**